Uncaught Error: Disallowed operation called within global scope with hono/jwt in CloudflareWorkers

### What version of Hono are you using?

4.0.2

### What runtime/platform is your app running on?

Cloudflare Workers

### What steps can reproduce the bug?

```js
import { Hono } from 'hono'
import { sign } from 'hono/jwt'

const app = new Hono()

app.get('/', async (c) => {
  const payload = {
    sub: 'user123',
    role: 'admin',
  }
  const secret = 'mySecretKey'
  const token = await sign(payload, secret)
  return c.json({ token }, 200)
})

export default app
```

The above code calls the sign function of ```hono/jwt``` and creates an endpoint that creates and returns a JWT token.

Run a local environment for ```CloudflareWorkers```.
The package manager is ```pnpm```.



### What is the expected behavior?

Ability to create JWT tokens by calling the ```sign``` function within Hono's handler.

### What do you see instead?

I ran the above code in the local environment of CloudflareWorkers and got the following error.

```
⎔ Starting local server...
[wrangler:inf] Ready on http://localhost:8787
✘ [ERROR] service core:user:carcharhinus: Uncaught Error: Disallowed operation called within global scope. Asynchronous I/O (ex: fetch() or connect()), setting a timeout, and generating random values are not allowed within global scope. To fix this error, perform this operation within a handler. https://developers.cloudflare.com/workers/runtime-apis/handlers/

    at null.<anonymous> (index.js:27321:29)


✘ [ERROR] MiniflareCoreError [ERR_RUNTIME_FAILURE]: The Workers runtime failed to start. There is likely additional logging output above.
```

It is suggested to use it in the handler instead of the global scope, but I am getting an error even though I am using it in a Hono handler.

### Additional information

The error about ```hono/jwt``` confirms that there is also an error about the JWT Auth Middleware.

I confirmed the same error as above when I run it in Cloudflare Workers local environment using jwt Middleware as per the official documentation.

```js
import { Hono } from 'hono'
import { jwt } from 'hono/jwt'

const app = new Hono()

app.use(
  '/auth/*',
  jwt({
    secret: 'it-is-very-secret',
  })
)

app.get('/auth/page', (c) => {
  return c.text('You are authorized')
})

export default app

```

--- a/deno_dist/middleware/jwt/index.ts
+++ b/deno_dist/middleware/jwt/index.ts
@@ -1,5 +1,5 @@
 import type { Context } from '../../context.ts'
-import { getCookie } from '../../helper.ts'
+import { getCookie } from '../../helper/cookie/index.ts'
 import { HTTPException } from '../../http-exception.ts'
 import type { MiddlewareHandler } from '../../types.ts'
 import { Jwt } from '../../utils/jwt/index.ts'

--- a/src/middleware/jwt/index.ts
+++ b/src/middleware/jwt/index.ts
@@ -1,5 +1,5 @@
 import type { Context } from '../../context'
-import { getCookie } from '../../helper'
+import { getCookie } from '../../helper/cookie'
 import { HTTPException } from '../../http-exception'
 import type { MiddlewareHandler } from '../../types'
 import { Jwt } from '../../utils/jwt'

